cmd/src/campaigns_apply.go

```diff
@@ -16,11 +16,11 @@ instance, creating or updating the described campaign if necessary.
 
 Usage:
 
-    src campaigns apply -f FILE -namespace NAMESPACE [command options]
+    src campaigns apply -f FILE [command options]
 
 Examples:
 
-    $ src campaigns apply -f campaign.spec.yaml -namespace myorg
+    $ src campaigns apply -f campaign.spec.yaml
```

Comment:
I think it might make sense to keep this in the examples:

```suggestion
    $ src campaigns apply -f campaign.spec.yaml
  
    $ src campaigns apply -f campaign.spec.yaml -namespace myorg
```